tests/subworkflows/nf-core/annotation/ensemblvep/test.yml

```diff
@@ -1,7 +1,11 @@
 - name: ensemblvep annotation_ensemblvep
   command: nextflow run ./tests/subworkflows/nf-core/annotation_ensemblvep -entry annotation_ensemblvep -c ./tests/config/nextflow.config -c ./tests/subworkflows/nf-core/annotation_ensemblvep/nextflow.config
   tags:
-    - annotation_ensemblvep
+    - subworkflows
+    - subworkflows/annotation/ensemblvep
```

Comment:
Can subworkflows have subfolders? I didn't take it into account in the new nf-core tools commands 😮

Reply:
Apparently. Ii makes sense though for `gatk` for example, I want to move those into their own directory. But we can put that off for now and just do something like `annotation_ => annotation/` in the future if we ever get around to it.